events/api/src/main/java/org/projectnessie/events/api/ReferenceEvent.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.events.api;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+
+/** Event that is emitted when a reference is created, updated or deleted. */
+public interface ReferenceEvent extends Event {
+
+  /** The name of the reference. */
+  String getReferenceName();
```

Comment:
Don't we have the "full reference name" here?

Reply:
I completely forgot it :-)